packages/amplify-category-storage/src/index.ts

```diff
@@ -75,9 +79,32 @@ export async function add(context: any, providerName: any, service: any) {
   return providerController.addResource(context, AmplifyCategories.STORAGE, service, options);
 }
 
-export async function console(context: any) {
-  printer.info(`to be implemented: ${AmplifyCategories.STORAGE} console`);
-}
+/**
+ * Open AWS Management Console for resources of storage category.
+ */
+export const console = async (context: $TSContext): Promise<void> => {
+  const { amplify } = context;
+  const amplifyMeta = stateManager.getMeta();
+  if (!amplifyMeta.storage || Object.keys(amplifyMeta.storage).length === 0) {
+    return printer.error('Storage has NOT been added to this project.');
```

Comment:
Since this function returns a void Promise, I think it's better if this line is broken up.

```suggestion
    printer.error('Storage has NOT been added to this project.');
    return;
```